code/modules/clothing/under/miscellaneous.dm

```diff
@@ -172,11 +172,13 @@
 	body_parts_covered = CHEST|GROIN
 
 /obj/item/clothing/under/misc/gear_harness/toggle_jumpsuit_adjust()
-	adjusted = !adjusted
-	if(adjusted)
-		body_parts_covered = NONE
-	else
+	if(!body_parts_covered)
+		to_chat(usr, "<span class='notice'>Your gear harness is now covering your chest and groin.</span>")
 		body_parts_covered = CHEST|GROIN
+	else
+		to_chat(usr, "<span class='notice'>Your gear harness is no longer covering anything.</span>")
+		body_parts_covered = NONE
+	return !body_parts_covered
```

Comment:
This line should probably always return true considering the return value is used to see if the body needs to be updated.

Reply:
Also done.